frontend/src/components/details/PieChartComponent.jsx

```diff
@@ -26,9 +32,19 @@ const PieChartDiagramComponent = () => {
             'Silicon steel',
             'Copper'
         ],
-        colors: ['#040f13', '#0b2d39', '#165a72', '#2596be', '#3ba1c5', '#00308F', '#d3eaf2'],
+        colors: [theme.color.TransformerOil, theme.color.Steel, theme.color.Pressboard, theme.color.StainlessSteel, theme.color.Alminium, theme.color.SiliconSteel, theme.color.Copper],
         fill: {
-            type: 'gradient'
+            type: 'gradient',
+            gradient: {
+                shade: 'dark',
+                type: "horizontal",
+                shadeIntensity: 0.5,
+                gradientToColors: undefined, // optional, if not defined - uses the shades of same color in series
+                inverseColors: true,
+                opacityFrom: 1,
+                opacityTo: 1,
+                stops: [0, 50, 100],
+                colorStops: []}
```

Comment:
@Waldleufer we do have "shadow effect" on the fonts and charts when using the cursor on them and now we do not have them there , I added this code to remove the shadowing.

Reply:
Cool! 

And what about the Tooltips? do you or maybe @gandompm have an Idea about that? Because the numbers don't match 100% and the `%` is missing in the tooltips

Reply:
@Waldleufer 
I do not see any problem with that. if you sum up all the percentages that are shown on each part, it would be 100%. 
also, the number on tooltips doesn't have a percentage notation because it is rounded down to an integer value.